projects/vdk-core/src/vdk/internal/core/errors.py

```diff
@@ -10,12 +10,7 @@
 """
 from __future__ import annotations
 
-import enum
-import logging
-import re
-from collections import defaultdict
-from enum import Enum
-from logging import Logger
+from vdk.internal.core.error_classifiers import *
```

Comment:
Please make sure this isn't imported anywhere else, e.g. plugins, other core components. There are also places in the code where you might encounter something like `errors.ResolvableBy.PLATFORM_ERROR`, so please check for those as well.

I'm not sure if the change will be backwards-compatible if the stuff you moved is imported in plugins. 😕

Reply:
As far as I know, when you move a class from one module to another but use an import statement in the original module to maintain its presence there (like using from .new_module import * in errors.py), for external code that imports package.errors.MyClass, there should be no change? I am not sure @antoniivanov what do you think?

Reply:
Yeah, that would work to keep backward compatible imports 

Please add a comment that indicate the purpose of this import . As we will get warnings from Codacy for unused imports so we need to know to keep it until it's needed.

Reply:
Also we should plan and change those occurrences where needed.

Reply:
I had no idea imports worked like that in python. Guess you learn something new every day. Thanks, guys!